client/rest-high-level/src/test/java/org/elasticsearch/client/CCRIT.java

```diff
@@ -260,7 +261,9 @@ public void testForgetFollower() throws IOException {
             final Map<?, ?> shardStatsAsMap = (Map<?, ?>) shardStats.get(0);
             final Map<?, ?> retentionLeasesStats = (Map<?, ?>) shardStatsAsMap.get("retention_leases");
             final List<?> leases = (List<?>) retentionLeasesStats.get("leases");
-            assertThat(leases, empty());
+            for (final Object lease : leases) {
+                assertThat(((Map<?, ?>) lease).get("source"), equalTo(ReplicationTracker.PEER_RECOVERY_RETENTION_LEASE_SOURCE));
```

Comment:
Not as robustly as I'd like, no. We could say there's no leases with source `"ccr"`, but that's a lot weaker than saying the only remaining leases are PRRLs, similarly to how we previously asserted that there were no leases at all.

Reply:
Can we use toMapExcludingPeerRecoveryRetentionLeases here?

Reply:
Not very easily. Here we are the other side of the high-level REST API, and this doesn't include indices stats so we don't have access to a `RetentionLeases` object. It would be quite some work to build one.

Reply:
ah, I did not realize that it's a rest test.